cfme/tests/intelligence/reports/test_widgets.py

```diff
@@ -221,3 +229,75 @@ def test_generate_widget_content_by_automate(request, appliance, klass, namespac
     widget.refresh()
     current_update = view.last_run_time.read().split(' ')[4]
     assert old_update != current_update
+
+
+@pytest.mark.meta(
+    automates=[1761836, 1753682],
+    blockers=[BZ(1761836, unblock=lambda context: context != ViaREST)],
+)
+@pytest.mark.tier(3)
+@test_requirements.rest
+@pytest.mark.parametrize("context", [ViaUI])
```

Comment:
Even though `ViaREST` is blocked right now, do we still want the test case to be parametrized on that?

Reply:
I knew something was wrong!!!!
ViaREST is supposed to be there, I had removed it while fixing the test failures. Thanks for catching that John!